ts/a11y/explorer.ts

```diff
@@ -145,17 +145,28 @@ export function ExplorerMathItemMixin<B extends Constructor<HTMLMATHITEM>>(
      * @param {ExplorerMathDocument} document The current document.
      */
     public attachExplorers(document: ExplorerMathDocument) {
-      this.attached = [];
```

Comment:
It looks like `this.attached` is no longer created or populated, but it is still used elsewhere in the code (in `rerender()` and `updateDocument()`).  Is that still correct?

Reply:
Good catch. 
I've now also made the more restarting more robust on collapsed elements. This only works with the latest fixes I made to SRE today.
There's still a minor kink to work out: When a collapse is triggered with the mouse while the key explorers are deeper in the tree they effectively refocus on a node that is no longer there. But that is for 3.2 to work out.
the refocus does not y

Reply:
Solved that problem now. I will make a small PR once this PR is merged.